package.json

```diff
@@ -9,9 +9,9 @@
     "eject": "rnstl && expo eject",
     "storybook": "start-storybook -p 7007",
     "build-storybook": "build-storybook",
-    "prettier": "prettier --write 'App/**/*.{ts,tsx,js}'",
+    "prettier": "prettier --write App/**/*.{ts,tsx,js}",
     "prestorybook": "rnstl",
-    "lint": "eslint './App/**/*.{js,ts,tsx}'"
+    "lint": "eslint ./App/**/*.{js,ts,tsx}"
```

Comment:
Are you sure that this will also work on MAC?

Reply:
@Mixxx-a idk but that's a way for windows. Probably should check it